### PR TITLE
TECH-174 feat: get mentor by availability

### PR DESCRIPTION
**Pull request by Andrew4Coding**
- Get mentor by availability

### CODE REVIEW

File: apps/api/v1/pages/api/availabilities/mentors/_get.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+import {
+  schemaAvailabilityCreateBodyParams,
+  schemaAvailabilityEditBodyParams,
+} from "~/lib/validations/availability";
+import { schemaUserReadPublic } from "~/lib/validations/user";
+
+async function getHandler(req: NextApiRequest) {
+  const {
+    body,
```
Comment (daniyusra) [inline apps/api/v1/pages/api/availabilities/mentors/_get.ts]: Why not use query parameters? Using body in a get request is unorthodox to say the least. 

Other API examples from the same subfolder to use query parameters are as follows

apps/api/v1/pages/api/availability/_get.ts
apps/api/v1/pages/api/bookings/_get.ts

**Comment by daniyusra** [comment]
The routing of the path seems to be confusing, not easy to access, and inconsistent with the design document. It might not be accessible at all. 

Rather than putting in the current path, which is 

`apps/api/v1/pages/api/availabilities/mentors/_get.ts`

I suggest putting it into the following directory

`apps/web/pages/api/availability`

With the following path

`apps/web/pages/api/availability/mentor-by-availabiltiy.ts`

Don't forget to make changes in the design document!

**Review by daniyusra — CHANGES_REQUESTED** [review]
The routing of the path seems to be confusing, not easy to access, and inconsistent with the design document. It might not be accessible at all.

Rather than putting in the current path, which is

apps/api/v1/pages/api/availabilities/mentors/_get.ts

I suggest putting it into the following directory

apps/web/pages/api/availability

With the following path

apps/web/pages/api/availability/mentor-by-availabiltiy.ts

Don't forget to make changes in the design document!

File: apps/web/pages/api/availabilities/mentor-by-availability.ts [hunk 2]
```diff
@@ -60,23 +61,36 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
...
-              },
-            },
-          },
-        },
       },
       take,
       skip,
     });
 
+    console.log(selectedUsers);
```
Comment (Andrew4Coding) [inline apps/web/pages/api/availabilities/mentor-by-availability.ts]: Overall udah bagus sih kak, maybe ini di remove aja console.log nya

**Pull request merged**